pkg/status/message/message_service_test.go

```diff
@@ -113,13 +109,53 @@ var _ = Describe("Config loader features", func() {
 				SetMatcher(ExpectPayload(toHaveModifiedBody)).
 				Reply(200)
 
-			hinter := ghservice.NewHinter(client, log.NewTestLogger(), commentsLoader, hintContext)
+			msgService := message.NewStatusMessageService(client, log.NewTestLogger(), commentsLoader, messageContext)
+
+			// when
+			err := msgService.StatusMessage(newBasicMsgCreator("New comment"), true)
+
+			// then
+			Ω(err).ShouldNot(HaveOccurred())
+		})
+
+		It("should modify existing comment when a new message is provided", func() {
```

Comment:
I am not sure if it is not confusing. We rarely modify our own comments to keep the context of the discussion right, so why would bot do that?